apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue

```diff
@@ -1,39 +1,25 @@
-<template>
-    <p-data-loader :loading="loading"
-                   class="cost-analysis-stacked-column-chart"
-    >
-        <template #loader>
-            <p-skeleton height="100%" />
-        </template>
-        <div ref="chartRef"
-             class="chart"
-        />
-    </p-data-loader>
-</template>
-
-<script lang="ts">
+<script lang="ts" setup>
 import {
-    getCurrentInstance,
-    reactive, toRefs, watch,
+    nextTick,
+    reactive, ref, watch,
 } from 'vue';
-import type { Vue } from 'vue/types/vue';
 
-import type { XYChart } from '@amcharts/amcharts4/charts';
-import * as am4charts from '@amcharts/amcharts4/charts';
-import * as am4core from '@amcharts/amcharts4/core';
+import type { TimeUnit } from '@amcharts/amcharts4/core';
```

Comment:
question: will this be replaced with v5 after?

Reply:
wow thanks😨